communities/london2.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "London Code Dojo",
+  "url": "https://www.meetup.com/London-Code-Dojo",
+  "location": {
+    "city": "London, United Kingdom",
+    "coordinates": [-0.12769666311382366, 51.50730924351245]
```

Comment:
Please change the coordinates to something like `[-0.09170726532940421,51.515753579776316]` while I think about how to support multiple communities in one city. Otherwise, only one dot will show up or be clickable at all.